docs/manual_install.rst

```diff
@@ -40,26 +40,19 @@ Bot Settings:
 Database
 --------
 
-Create a new mysql database and user for your bot.
+Create a new mysql database and user for your bot, granting full priviledges for the db. For example:
 
-Command to access the terminal as root user: ``mysql -u root -p``
+.. code-block:: shell
+    mysql -u root -p
+    MariaDB [(none)]> CREATE USER 'username'@'localhost' IDENTIFIED BY 'PASSWORD';
+    MariaDB [(none)]> CREATE DATABASE databasename;
+    MariaDB [(none)]> GRANT ALL PRIVILEGES ON databasename.* TO 'username'@'localhost';
+    MariaDB [(none)]> FLUSH PRIVILEGES;
+    MariaDB [(none)]> exit
 
-Command to create a user with localhost access (Only allow localhost access!): ``CREATE USER 'USERNAME'@'localhost' IDENTIFIED BY 'PASSWORD';``
+The DB structure is imported the first time the bot is used. If it fails for any reason you can also run it manually:
```

Comment:
Should this be also menationed in docker side of documentation?

Reply:
Done